AWS Publish Images.yml

```diff
@@ -980,6 +980,7 @@ stages:
               3. Even if only the images are being updated, the Template CD pipeline must be run to upload the updated templates to S3 ready for publication.
               4. Increment the Version Numbers in the Build Image Release Artefacts CI pipeline
               5. Increment the VersionText-TemplateCI variable for the Template CI pipeline testing
+            onTimeout: 'resume'
```

Comment:
Resume? Does this mean the next stage runs? That is not correct. It should fail on timeout. It can always be re-run through 'run all failed jobs'.